.github/utils/pyproject_to_requirements.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import sys
+from pathlib import Path
+import toml
+
+def main(pyproject_path: Path, exclude_optional_dependencies: bool = False):
+    content = toml.load(pyproject_path)
+    deps = set(content["project"]["dependencies"])
+
+    if not exclude_optional_dependencies:
+        optional_deps = content["project"].get("optional-dependencies", {})
+        for dep_list in optional_deps.values():
+            deps.update(dep_list)
+
+    sys.stdout.write("\n".join(sorted(deps)) + "\n")
```

Comment:
Was `print` not good?

Reply:
copy-paste from Haystack script.
Now I'm using `print` and it works :-)